src/testbase.rs

```diff
@@ -166,7 +163,8 @@ pub fn traverse_some_bh<BH: BoundingHierarchy>() {
 }
 
 /// A triangle struct. Instance of a more complex `Bounded` primitive.
-#[derive(Debug)]
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "serde_impls", derive(serde::Serialize, serde::Deserialize))]
 pub struct Triangle {
```

Comment:
I feel like this should be removed since this pr includes a triangle shape, and having 2 public triangle structs is confusing.